src/Math-Core-Distribution/PMMultivariateNormalDistribution.class.st

```diff
@@ -0,0 +1,98 @@
+Class {
+	#name : #PMMultivariateNormalDistribution,
+	#superclass : #PMProbabilityDensity,
```

Comment:
is a multivariate normal distribution also a distribution? I'm not sure for example that ``kurtosis`` or ``skewness`` apply in this context.